src/VolleyManagement.Backend/VolleyManagement.UI/App_Start/BundleConfig.cs

```diff
@@ -31,7 +31,9 @@ private static void RegisterCommonBundles(BundleCollection bundles)
         {
             bundles.Add(new StyleBundle("~/bundles/commoncss")
                 .Include(
-                    "~/Content/bootstrap.min.css"));
+                    "~/Content/bootstrap.min.css")
+                .Include(
+                    "~/Content/Site.css"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code not to use hardcoded absolute paths or URIs. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1075)

Reply:
You can ignore this error